gcip/core/job.py

```diff
@@ -21,16 +21,16 @@
 #   script: build my artifact
 ```
 
-A `Job` has always a `script` and at least one of `namespace` and `name`.
-The `namespace` will be used for the name of the stage of the job and the
+A `Job` has always a `script` and at least one of `stage` and `name`.
+The `stage` will be used for the name of the stage of the job and the
 job name itself, whereas `name` is only used for the job`s name. When adding
 a job to a `gcip.core.pipeline.Pipeline` or a `gcip.core.sequence.Sequence`
-you can and should define a `name` or `namespace` too. This is how you
+you can and should define a `name` or `stage` too. This is how you
```

Comment:
```suggestion
you have to define a `name` or `stage`. This is how you
```

Reply:
No, that is quite fine. You should not simply add a name or stage. You should additionally add a name or namespace to the job when added - additional to the name or namespace of the job itself. So you should add it too.